### PR TITLE
The Apple page clone project 

### PR DESCRIPTION
**Pull request by RNtaate**
In this project, I created a clone of an Apple web page, used gradient and background images in styling the backgrounds of most of the HTML elements used in the project.

### CODE REVIEW

**Review by sumancrest0001 — COMMENTED** [review]
STATUS: CHANGES REQUIRED
Impressive work @RNtaate 
Most of the requirements are ticked and you are very close to getting approval for this project. Let's fix the following issues and make it better.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <div class="lower-sec-div">
+        <div class="phone-type-div phone1">
+        </div>
+        <div class="phone-type-div phone2">
+        </div>
+        <div class="phone-type-div phone3">
+        </div>
+        <div class="phone-type-div phone4">
+        </div>
+    </div>
```
Comment (sumancrest0001) [inline index.html]: Let's add a link to each article as in the [original](https://web.archive.org/web/20140301004610/http://www.apple.com/) one.

File: styler.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+}
+
+.left-footer-div > p > a {
+  margin: 0 5px;
+}
+
+.right-footer-div > p > a {
+  margin: 0 5px;
+}
+
```
Comment (sumancrest0001) [inline styler.css]: We have to clean up the CSS file and remove all the repeated code. Let's follow the DRY principle.

File: styler.css [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+.every-div {
+  margin-bottom: 20px;
+}
+
+.every-div > a > img {
+  width: 100%;
+}
+
```
Comment (sumancrest0001) [inline styler.css]: We have to clean up the CSS file and remove all the repeated code. Let's follow the DRY principle.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
Kindly make the request changes and submit another review form. I am sure that it won't take a long time to make these changes.

Need help or have a query? Contact me on slack **Suman Shrestha**
**Happy coding**

**Comment by RNtaate** [comment]
Removed all repeated code from the CSS file and added a link to each article in the lower-sec-div just as in the original one.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project. :fireworks:  Keep it up.
##
### The [clone](https://raw.githack.com/RNtaate/roy-apple-page-project/apple-b/index.html) shows good understanding and usage of building with backgrounds and gradients. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
-  The project has a descriptive README.md and is professionally done. Good job with that, you just have to update the live link to match the changes and you should be good.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @RNtaate! :+1:
- Carry out the change with the live link and then merge this. You don't have to submit another code review. I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**